File: generateclasses/Stylematiere_v.cs
namespace entite;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

[Table("stylematiere_v")]
public class Stylematiere_v : Motherobj<Stylematiere_v>  {
    [Column("idstyle")]
    public int idstyle {get; set;} 
    [Column("nomstyle")]
    public string nomstyle {get; set;} 
    [Column("idmatiere")]
    public int idmatiere {get; set;} 
    [Column("nommatiere")]
    public string nommatiere {get; set;} 
    [Column("idunite")]
    public int idunite {get; set;} 
    [Column("unite")]
    public string unite {get; set;} 

    public Stylematiere_v(){ }
    

}

File: generateclasses/Stylematiere_v.java
package entite;

public class Stylematiere_v{
    int idstyle; 
    String nomstyle; 
    int idmatiere; 
    String nommatiere; 

    public Stylematiere_v(){ }
    
    public int getIdstyle(){
        return this.idstyle;
    }
    public void setIdstyle(int idstyle){
        this.idstyle=idstyle;
    }
    public String getNomstyle(){
        return this.nomstyle;
    }
    public void setNomstyle(String nomstyle){
        this.nomstyle=nomstyle;
    }
    public int getIdmatiere(){
        return this.idmatiere;
    }
    public void setIdmatiere(int idmatiere){
        this.idmatiere=idmatiere;
    }
    public String getNommatiere(){
        return this.nommatiere;
    }
    public void setNommatiere(String nommatiere){
        this.nommatiere=nommatiere;
    }

}
